app/services/slack/announcer.rb

```diff
@@ -0,0 +1,31 @@
+module Slack
+  # a thin wrapper on Slack::Notifier
+  # see <https://github.com/stevenosloan/slack-notifier> for additional options
```

Comment:
```suggestion
  # for additional options, see https://github.com/stevenosloan/slack-notifier
```
nitpick but i think this is a bit cleaner